tests/Feature/Controller/User/RegisterUserControllerTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
```

Comment:
declare(strict_types=1);

Also, in order to maximize the functionality provided by strict_types=1, it's advisable to specify return types and argument data types whenever possible.

Reply:
Got it